client/src/Components/InstagramPage/InstagramPage.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import "./InstagramPage.css";
+import Post from "./Post/Post";
```

Comment:
Please modify the module path as if we are creating the folder InstagramPage in components then it will create an error;   modify it as " . . /Post/Post"